examples/multiomics_mogonet/config.py

```diff
@@ -0,0 +1,47 @@
+from yacs.config import CfgNode
+
+# ---------------------------------------------------------
+# Config definition
+# ---------------------------------------------------------
+
+C = CfgNode()
+
+# ---------------------------------------------------------
+# Dataset
+# ---------------------------------------------------------
+C.DATASET = CfgNode()
+C.DATASET.ROOT = "dataset/"
+C.DATASET.NAME = "TCGA_BRCA"
+C.DATASET.URL = "https://github.com/pykale/data/raw/main/multiomics/TCGA_BRCA.zip"
+C.DATASET.RANDOM_SPLIT = False
+C.DATASET.NUM_VIEW = 3
+C.DATASET.NUM_CLASS = 5
+
+# ---------------------------------------------------------
+# Solver
+# ---------------------------------------------------------
+C.SOLVER = CfgNode()
+C.SOLVER.SEED = 2023
+C.SOLVER.MAX_EPOCHS_PRETRAIN = 500
+C.SOLVER.MAX_EPOCHS = 2500
```

Comment:
These are a lot of epochs. Have you considered to make the default a faster version with fewer epochs?

Reply:
I have added more information about the default values in the config file.